vcd/resource_vcd_vapp.go

```diff
@@ -486,6 +552,35 @@ func resourceVcdVAppRead(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf("unable to set guest properties in state: %s", err)
 	}
 
+	// Power status is not easy to define.
+	// It should be set when status == 4, but even when we request it the status
+	// change may not happen until late. Returning the power status at an early
+	// stage may mislead both users and terraform, with the result of having
+	// unnecessary update requests.
+	// We have two fields that produce better information:
+	// * status (numeric status code, such as "4")
+	// * status_text (status as a string, such as "POWERED_ON")
+	// _ = d.Set("power_on", vapp.VApp.Status == 4)
```

Comment:
this line of comment isn't needed anymore?

Reply:
I left it on purpose, to point that we're talking about this property